discovery/discovery.go

```diff
@@ -10,23 +14,90 @@ import (
 	"github.com/golang/glog"
 )
 
-type offchainOrchestrator struct {
-	uri   *url.URL
+const GetOrchestratorsTimeoutLoop = 3 * time.Second
+
+type offchainOrchestrators struct {
```

Comment:
Should we find another name for this?  `offchainOrchestrators` seems less appropriate than something like `orchestratorUrls` (it's especially strange to return `offchainOrchestrators` for `NewOnchainOrchestratorPool`

Reply:
Ah yes, I was looking at that and thought it strange too, but hadn't done anything about it. Would orchestratorPool make more sense? What do you think @j0sh

Reply:
Sure, either `orchestratorPool` or `orchestratorUrls` works

Reply:
Awesome! I like `orchestratorPool`, since we already have `NewOnchainOrchestratorPool`, and `NewOrchestratorPool`